---
permalink: /
title: "Sahl Lab for Understanding Infectious Disease GEnomics [SLUDGE](Sludge.png)"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

{% include base_path %}

### Overview

My lab is located in Flagstaff, Arizona, USA, at 7000ft (2137m) above sea level. In my group, we incorporate both wet
lab techniques and bioninformatics methods to better understand pathogen ecology, antimicrobial resistance, and 
pathogenesis. A list of current projects ongoing in the laboratory include:

**1**. *Antimicrobial resistance mechanism identification in Acinetobacter baumannii*.
2. **Characterization of Escherichia coli in Urinary Tract Infections**.
3. **

### Software

My lab has developed or helped in the development of the following software applications:

1. Phylomark. A method to identify phylogenetic markers from whole genome alignments. Although originally designed
for whole genome alignments, but has now been adapted to work with SNP data from NASP. The software repository
is located [here](https://github.com/jasonsahl/Phylomark). The goal is to find good phylogenetic markers that
recapitulate the core genome phylogeny.  
2. LS-BSR. A method to compare the gene content between bacterial genomes. A  link to the paper is [here](https://peerj.com/articles/332/)
The software repository is located [here](https://github.com/jasonsahl/LS-BSR)  
3. NASP. 
4. WG-FAST.  