---
permalink: /
title: "Sahl Lab for Understanding infectious Disease GEnomics" 
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

{% include base_path %}

### Overview

My lab is located in Flagstaff, Arizona, USA, at 7000ft (2137m) above sea level. In my group, we incorporate both wet
lab techniques and bioninformatics methods to better understand pathogen ecology, antimicrobial resistance, and 
pathogenesis. A list of current projects ongoing in the laboratory include:

1. *Antimicrobial resistance (AMR) mechanism identification in Acinetobacter baumannii*. A. baumannii has a highly plastic
 genome where AMR mechanisms are not broadly conserved. We are using a paired phylogenetic approach to identify novel
 mechanisms of resistance in this globally important AMR pathogen.  
2. *Characterization of Escherichia coli in Urinary Tract Infections (UTIs)*. UTIs are a major problem in the United
States with most infections attributed to E. coli. We are using new Amplicon Sequencing methods to characterize E. coli 
directly from urine, providing information on AMR and strain level identity that can be used for source attribution  
3. *Comparative genomics of Burkholderia pseudomallei*. We have published papers on the identification of diagnostic targets
 in B. pseudomallei using comparative genomics []. We are also interested in genotype/phenotype relationships in B. pseudomallei
 using genome wide association studies (GWAS) as well as gene knock out experiments  
4. *Development of Amplicon Sequencing projects for strain level resolution*. While high throughput sequencing is becoming 
commonplace in most research laboratories, cheap and rapid assays that provide maximum resolution are still needed. We are
 investigating methods to provide strain level resolution from multiple amplicons in a single PCR reaction, which will 
 provide rapid results at a reasonable price.  
5. *Source tracking of Clostridioides difficile*. C. difficile is regarded as the most common hospital acquired pathogen 
and is associated with serious disease. We are sequencing strains from the clinic, dogs, and the environment in order to
 help identify how patients are being infected.  

### Software

My lab has developed or helped in the development of the following software applications:

1. *Phylomark*. A method to identify phylogenetic markers from whole genome alignments. Although originally designed
for whole genome alignments, but has now been adapted to work with SNP data from NASP. The software repository
is located [here](https://github.com/jasonsahl/Phylomark). The goal is to find good phylogenetic markers that
recapitulate the core genome phylogeny.  
2. *LS-BSR*. A method to compare the gene content between bacterial genomes. A  link to the paper is [here](https://peerj.com/articles/332/)
. The software repository is located [here](https://github.com/jasonsahl/LS-BSR)  
3. *NASP*. Our method for SNP discovery that supports reads as well as genome assemblies. A link to the paper is [here](http://mgen.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000074).
  The software repository is located [here](https://github.com/TGenNorth/NASP)  
4. *WG-FAST*. A method to place isolates with partial SNP genotypes into a reference phylogeny. A link to the paper is [here](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-015-0176-9).
  The software repository is located [here](https://github.com/jasonsahl/wgfast)  
5. *UGAP*. A wrapper to run SPAdes, polish the assembly with PILON, and report stats that may be useful for identifying
potential contaminants. The software repository is located [here]().

### People  

Current members of my laboratory include:  

1. Chase Williamson, Ph.D. Chase is a Research Associate and works on various bioinformatics projects and has published multiple papers, primarily
on Clostridium botulinum.  
2. Adam Vazquez. Adam is a Research Specialist who works in the laboratory, but also performs data analyses on all active projects.  
3. Chandler Roe.
