---
permalink: /
title: "Sahl Lab for Understanding Infectious Disease GEnomics" 
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

{% include base_path %}

[logo]: https://github.com/jasonsahl/jasonsahl.github.io/tree/master/images/Sludge.png "The Sludge"

### Overview

My lab is located in Flagstaff, Arizona, USA, at 7000ft (2137m) above sea level. In my group, we incorporate both wet
lab techniques and bioninformatics methods to better understand pathogen ecology, antimicrobial resistance, and 
pathogenesis. A list of current projects ongoing in the laboratory include:

1. *Antimicrobial resistance (AMR) mechanism identification in Acinetobacter baumannii*. A. baumannii has a highly plastic
 genome where AMR mechanisms are not broadly conserved. We are using a paired phylogenetic approach to identify novel
 mechanisms of resistance in this globally important AMR pathogen.  
2. *Characterization of Escherichia coli in Urinary Tract Infections*.
3. *Comparative genomics of Burkholderia pseudomallei*.
4. *Development of Amplicon Sequencing projects for strain level resolution*.

### Software

My lab has developed or helped in the development of the following software applications:

1. *Phylomark*. A method to identify phylogenetic markers from whole genome alignments. Although originally designed
for whole genome alignments, but has now been adapted to work with SNP data from NASP. The software repository
is located [here](https://github.com/jasonsahl/Phylomark). The goal is to find good phylogenetic markers that
recapitulate the core genome phylogeny.  
2. *LS-BSR*. A method to compare the gene content between bacterial genomes. A  link to the paper is [here](https://peerj.com/articles/332/)
. The software repository is located [here](https://github.com/jasonsahl/LS-BSR)  
3. *NASP*. Our method for SNP discovery that supports reads as well as genome assemblies. A link to the paper is [here](http://mgen.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000074).
  The software repository is located [here](https://github.com/TGenNorth/NASP)  
4. *WG-FAST*. A method to place isolates with partial SNP genotypes into a reference phylogeny. A link to the paper is [here](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-015-0176-9).
  The software repository is located [here](https://github.com/jasonsahl/wgfast)  
5. *UGAP*.  